docs/verein360-enpoint.md

```diff
@@ -0,0 +1,158 @@
+# Verein360 Endpoint Documentation
+
+## Introduction
+Verein360/BLSV should be able to send already approved entitlementcard application for their trainers.
+An api token that can be generated from the "Digitale Druckerei" account is required to access the endpoint.
+All details about the endpoint can be found in the [API Documentation](../specs/backend-api.graphql).
+
+## Usage
+
+### Set Authourization Header
+
+Set the HTTP header `Authorization Bearer <api-token>` to access the endpoint.
+
+### Call GraphQL Mutation to add an application
+
+```graphql
+mutation addEakApplication($regionId: Int!, $application: ApplicationInput!, $project: String!) {
+  result: addEakApplication(regionId: $regionId, application: $application, project: $project)
+}
+```
+
+### Sample Mutation Variables
+
+```json
+  {
+  "regionId": 2,                                      // Must be the correct region ID where the card owner lives
+  "application": {
+    "applicationDetails": {
+      "applicationType": "FIRST_APPLICATION",         // Must be FIRST_APPLICATION
+      "cardType": "BLUE",                             // Must be BLUE
+      "givenInformationIsCorrectAndComplete": true,   // Must be true
+      "hasAcceptedEmailUsage": true,                  // true or false
+      "hasAcceptedPrivacyPolicy": true,               // Must be true
+      "wantsDigitalCard": true,                       // Must be true
+      "wantsPhysicalCard": false,                     // Must be false
+      "blueCardEntitlement": {
+        "entitlementType": "WORK_AT_ORGANIZATIONS",   // Must be WORK_AT_ORGANIZATIONS
+        "workAtOrganizationsEntitlement": {
+          "list": {
+            "amountOfWork": 7.5,
+            "organization": {
+              "address": {
+                "country": {
+                  "shortText": "Germany"
```

Comment:
:question: Why are these `shortText` attributes needed?
Not sure about this one:
```suggestion
                  "shortText": "Deutschland"
```

Reply:
if you send an application you can check the actual payload. But good question why this was implemented like this, maybe @michael-markl can give some input